react/Gallery.js

```diff
@@ -169,7 +168,7 @@ Gallery.propTypes = {
 Gallery.defaultProps = {
   maxItemsPerPage: 10,
   products: [],
-  itemWidth: 300,
+  itemWidth: 285,
```

Comment:
@oldufre explained that the item width should be dynamic. Has it changed?

Reply:
Not really, but I will do this in other pr. This is only to fix IE grid compatibility.